aqua/aquaterm/Portfile

```diff
@@ -59,6 +59,12 @@ post-patch {
         ${worksrcpath}/adapters/pgplot/ReadMe
 }
 
+# this port does not build with the new xcode build system at present
+if {[vercmp ${xcodeversion} 10.0] >= 0} {
+    build.pre_args      -UseNewBuildSystem=NO
```

Comment:
I recommend patching rather than using the old build system for the build phase. We're going to have to bite the bullet eventually.